Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -3,4 +3,11 @@ module Superenv
   def self.bin
     (HOMEBREW_SHIMS_PATH/"linux/super").realpath
   end
+
+  def determine_rpath_paths
+    paths = ["#{HOMEBREW_PREFIX}/lib"]
+    paths += deps.map { |d| d.opt_lib.to_s }
```

Comment:
I haven't yet fixed this for the `rpath` PR for `--env=std`. There's shared code, so I think I'll merge this PR first, and then rebase the `rpath` code on top of this merged PR.